git_machete/utils.py

```diff
@@ -94,7 +94,21 @@ def debug(msg: Optional[str] = None) -> None:
     if debug_mode:
         function_name = bold(inspect.stack()[1].function)
         args, _, _, values = inspect.getargvalues(inspect.stack()[1].frame)
-        function_args = bold(f"({', '.join([arg+'='+str(values[arg]) if arg != 'self' else 'self' for arg in args])})")
+
+        redact_arg_by_name = ['token']
```

Comment:
While you're here, I'd also add:
```suggestion
        redact_arg_by_name = ['access_token', 'password', 'secret', 'token']
```
to make it future-proof

Reply:
Actually, it can be a set, same for `redact_arg_by_value` below